src/components/product/OutOfStock/OutOfStock.tsx

```diff
@@ -27,20 +23,25 @@ export interface OutOfStockProps {
   buttonText?: string
   /**
    * Icon displayed inside the button.
```

Comment:
Looks like the comments are switched hahah.
```suggestion
   * Message describing when the user will be notified.
```